scalyr_agent/configuration.py

```diff
@@ -699,6 +701,42 @@ def __get_default_hostname(self):
             result = result.split(".")[0]
         return result
 
+    @staticmethod
+    def get_session_ids_of_the_worker(worker_config):  # type: (Dict) -> List
+        """
+            Generate the list of IDs of all sessions for the specified worker.
+            :param worker_config: config entry for the worker.
+            :return: List of worker session IDs.
+        """
+        result = []
+        for i in range(worker_config["sessions"]):
+            # combine the id of the worker and session's position in the list to get a session id.
+            worker_session_id = "%s_%s" % (worker_config["id"], i)
```

Comment:
Do we have some kind of regex validation for valid characters in the worker id config option?

If not, it's probably a good idea to only allow ``a-zA-Z-`` or similar.

Technically, I believe we use ``os.path.join`` everywhere so using arbitrary characters should still be safe from command / shell injections perspective, but it's probably still better we limit it (at the very list it could cause issues if we split the string and the id contains ``_`` character).